compiler/main/driver.cpp

```diff
@@ -308,6 +309,7 @@ int fGPUBlockSize = 0;
 char fCUDAArch[16] = "sm_60";
 
 chpl::Context* gContext = nullptr;
+std::vector<std::pair<std::string,std::string>> gDynoParams;
```

Comment:
let's make this `static` since it's only used within this file (during command line argument parsing). The result is saved in the gContext tables.

Reply:
it's also used in config.cpp though, where it gets populated